src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,199 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tale;
```

Comment:
```suggestion
    private Node<T> tail;
```
fix correct spelling, please